deploy/static/provider/scw/deploy.yaml

```diff
@@ -673,6 +675,8 @@ spec:
               valueFrom:
                 fieldRef:
                   fieldPath: metadata.namespace
+          securityContext:
```

Comment:
This is new. Let's keep an eye on errors on this

Reply:
yea that is new....the generate script must of not liked something